bigbluebutton-html5/imports/ui/components/chat/message-list/component.jsx

```diff
@@ -78,14 +77,12 @@ class MessageList extends Component {
       return;
     }
 
-    const { scrollArea } = this.refs;
```

Comment:
It would be much easier if you changed `const { scrollArea } = this.refs;` to `const { scrollArea } = this;` rather than putting `this.` in front of every use of `scrollArea`